src/node.cc

```diff
@@ -2379,6 +2415,20 @@ void OnMessage(Local<Message> message, Local<Value> error) {
 }
 
 
+void ClearFatalExceptionHandlers(Isolate* isolate) {
```

Comment:
Also, is there a reason this function takes a `v8::Isolate*` instead of a `node::Environment*`?  All call sites appear to be calling it with `env->isolate()`, after which this function looks up the environment again with `Environment::GetCurrent(isolate)`.